en/views/themes.rst

```diff
@@ -43,6 +43,14 @@ If a view file can't be found in the theme, CakePHP will try to locate the view
 file in the **src/Template/** folder. This way, you can create master template files
 and simply override them on a case-by-case basis within your theme folder.
 
+Now download a new plugin for your theme. The plugin is loaded in **config/bootstrap.php**.
```

Comment:
Where is the reader downloading the theme from?

Reply:
Yes

Reply:
Do you understand my idea?

Reply:
Yes. I'll tweak this a bit.